db/migrate/20160226092911_separate_openstack_network_manager_from_openstack_cloud_manager.rb

```diff
@@ -33,6 +33,8 @@ def affected_classes
 
   def up
     # Separate NetworkManager from CloudManager and move network models under NetworkManager
+    ExtManagementSystem.connection.schema_cache.clear!
```

Comment:
@Ladas is this line needed?  I thought only the reset_column_information was needed.